tools/harness-thci/OpenThread.py

```diff
@@ -3492,14 +3492,16 @@ def stopListeningToAddr(self, sAddr):
         return True
 
     @API
-    def registerMulticast(self, sAddr='ff04::1234:777a:1', timeout=MLR_TIMEOUT_MIN):
+    def registerMulticast(self, listAddr=['ff04::1234:777a:1'], timeout=MLR_TIMEOUT_MIN):
```

Comment:
```suggestion
    def registerMulticast(self, listAddr=('ff04::1234:777a:1', ), timeout=MLR_TIMEOUT_MIN):
```
nit: avoid mutable default arguments